<launch>
  <include file="$(find clbrobot)/launch/navigate.launch" />
  
  <!-- startup patrol navigation node -->
  <node pkg="clbrobot" type="clbrobot_patrol_nav.py" name="clbrobot_patrol_nav_node" output="screen" >
    <param name="rest_time"     value="5" />
    <param name="keep_patrol"   value="true" />
    <param name="random_patrol" value="false" />
    <param name="patrol_type"   value="0" />
    <param name="patrol_loop"   value="1" />
    <param name="patrol_time"   value="3" />
  </node>
 
</launch>

